pkgs/development/python-modules/serversyncstorage/default.nix

```
@@ -38,6 +37,7 @@ buildPythonPackage rec {
   ];
 
   meta = with stdenv.lib; {
+    broken = !(isPy27 && cornice.version == "0.17");
```

It seems like syncserver builds just fine with cornice overridden to 0.20.0 as well. So this should probably be
```suggestion
    broken = lib.versionAtLeast cornice.version "1.0.0";
```
@Nadrieril Why did you pin it to 0.17?

---

I don't quite remember. I think it clashed with some other dependency of syncserver. If you can build and run syncserver without this constraint then you can definitely remove it

---

I have no way to test syncserver. It would nice if you could do so with version 0.20.0 of cornice.

---

Ah, I found the original reason: `serversyncstorage` required cornice v0.17 and did not work with any later version (see https://github.com/NixOS/nixpkgs/pull/54739).
And [it looks like it is still the case](https://github.com/mozilla-services/server-syncstorage/blob/e7946d1f73369a431e8743d8e10387385f7d812a/requirements.txt#L6)

---

I checked, `serversyncstorage` has barely changed since https://github.com/NixOS/nixpkgs/pull/54739, so it will still require cornice 0.17. And having different versions of cornice in the dependency graph breaks, that was the reason for https://github.com/NixOS/nixpkgs/pull/54739. So I'm afraid we really do need to pin the older version of cornice.

---

Ok, in that case we can use `broken = cornice.version != "0.17"`.

---

Yeah, I simply wasn't using my whole brain when I moved the whole clause to `broken`. Pushing a fixed version...